src/applications/claims-status/containers/StemClaimStatusPage.jsx

```diff
@@ -50,17 +48,19 @@ class StemClaimStatusPage extends React.Component {
       );
     }
 
+    const crumb = {
+      href: `your-stem-claims/${params.id}`,
+      label: 'Your Rogers STEM Scholarship application status details',
+      isRouterLink: true,
+    };
+
     return (
       <div>
         <div name="topScrollElement" />
         <div className="vads-l-grid-container large-screen:vads-u-padding-x--0">
           <div className="vads-l-row vads-u-margin-x--neg1p5 medium-screen:vads-u-margin-x--neg2p5">
             <div className="vads-l-col--12">
-              <ClaimsBreadcrumbs>
-                <Link to={claimsPath}>
-                  Your Rogers STEM Scholarship application status details
-                </Link>
-              </ClaimsBreadcrumbs>
+              <ClaimsBreadcrumbs crumbs={crumb} />
```

Comment:
I think this needs to be wrapped in square brackets (`[`), right?

Reply:
Actually, technically it works now because you don't need them, you can call `concat` with a value instead of an array and it will just append the value to the array. So I guess as long as we do it one way or the other for all of them than that is fine